docs/docs/adding-page-transitions-with-plugin-page-transitions.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Adding Page Transitions with gatsby-plugin-page-transitions
+---
+
```

Comment:
Could you please add one or two paragraphs explaining what this doc would be about, e.g. like in [these commits](https://github.com/gatsbyjs/gatsby/pull/12779/commits/be18fd60f63bc14d7c980ced8a483d38400ba243) or e.g. how Wikipedia displays their stubs.

So that it's not just "hey, this is a stub" without context. But already a bit of content (for the people searching for help) and some directions on what to write (for the people who want to expand this stub).

Thanks!

Reply:
I've added a bit about ideally what would go in that doc/guide!